### PR TITLE
Shopify to Salesforce - Orders

### PR DESCRIPTION
**Pull request by FrendsSarlinS**
#99 

Checklist for review:
- [x] Process name is fine, uses normal casing and descriptive
- [x] Process description is fine, uses normal casing, dots and is descriptive
  - [x] Element names use normal casing, no unneeded capital letters.
- [x] Correct casing used for formats and terms, e.g. CSV, FTP, JSON, Avro
  - [x] Element names do not contain dots
  - [x] Comments / note element text are in good language and end with dots.
- [x] Template parameters checked
  - [x] Parameter descriptions are good and end with dots.
  - [x] Sensitive parameters are marked as secrets
- [x] "Turn off logging" switch is on for any task that has sensitive data as output
- [x] End "balls" like error throws and return statements have titles (double-click) and reasonable information.
  - [x] Results have good summaries, preferably with statistics of some sort like "processed X files"
  - [x] Errors have some error info, usually from task (e.g. some ErrorMessage property contains additional error info)
- [x] Process is organised with groups (dashed boxes)
- [x] Straight lines are preferred where possible
- [x] Subprocesses are forbidden
- [x] Text mode is preferred over Expressions in expressions, e.g. instead of *$"This is {#var.error}"* use *This is {{#var.error}}*
- [x] Process metadata file is present and content is fine (version number and name)
- [x] Process export file is present (process.json)
- [x] Process description file is present (all sections filled in)
- [x] Process description has a SVG screenshot of the process
- [x] Tempalte files are going to the correct folder

## Summary by CodeRabbit

- **New Features**
  - Introduced a template for exporting sales orders from Shopify and importing them into Salesforce, ensuring data consistency and uniqueness.
  - Implemented error handling to stop processing on errors and display them at the end of the process, with optional retries for transient connection errors.

### CODE REVIEW

**Review by jannejjj — COMMENTED** [review]
The template works correctly, just add titles to the returns and change the mode from expression to text in the throw. 👌

**Comment by FrendsSarlinS** [comment]
New commit is in!

**Review by jannejjj — DISMISSED** [review]
After the latest changes, everything looks OK!

File: Templates/E-Commerce/Shopify/Shopify to Salesforce - Orders/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Shopify to Salesforce - Orders",
+    "version": "1.0.0",
+    "tags": [],
+    "additionalSystemsUsed": [
```
Comment (jannejjj) [inline Templates/E-Commerce/Shopify/Shopify to Salesforce - Orders/metadata.json:5]: Mention Shopify here

File: Templates/E-Commerce/Shopify/Shopify to Salesforce - Orders/long-description.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+This template exports sales orders from Shopify and imports them to Salesforce.
```
Comment (jannejjj) [inline Templates/E-Commerce/Shopify/Shopify to Salesforce - Orders/long-description.md:1]: Newline here and other places in the file 😄

**Comment by FrendsSarlinS** [comment]
New commit is in with the edits to description and metadata files!

**Review by jannejjj — APPROVED** [review]
The description and metadata were corrected, and the template has already been tested to work previously - we're done!

**Pull request merged**